python/coinbase-agentkit/coinbase_agentkit/action_providers/wallet/schemas.py

```diff
@@ -13,3 +16,26 @@ class GetBalanceInput(BaseModel):
 
     # No additional fields needed as this action doesn't require any input parameters
     pass
+
+
+class NativeTransferInput(BaseModel):
+    """Input schema for native asset transfer."""
+
+    to: str = Field(..., description="The destination address to transfer to (e.g. '0x5154eae861cac3aa757d6016babaf972341354cf')")
+    value: str = Field(..., description="The amount to transfer in whole units (e.g. '1.5' for 1.5 ETH)")
+
+    @field_validator("to")
+    @classmethod
+    def validate_address(cls, v: str) -> str:
+        """Validate the Ethereum address."""
+        try:
+            Web3.to_checksum_address(v)
+            return v
+        except ValueError as e:
+            raise ValueError("Invalid Ethereum address format") from e
```

Comment:
we cannot valid this as checksummed hex string address. It does not have that requirement on other protocol families. Please remove all validation on this field other than that it is required @stat